src/SwapStep.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SubscriptionModels } from 'azure-arm-resource';
+// tslint:disable-next-line:no-require-imports
+import WebSiteManagementClient = require('azure-arm-website');
+import { IAzureNode, UserCancelledError } from 'vscode-azureextensionui';
+import { DeploymentSlotTreeItem } from './explorer/DeploymentSlotTreeItem';
+import { IQuickPickItemWithData } from './util';
+import { nodeUtils } from './utils/nodeUtils';
+import { WizardBase, WizardStep } from './wizard';
+
+export class SwapStep extends WizardStep {

```

Comment:
nit: If we're moving this to another file, I would prefer to name it 'SwapSlotStep' (otherwise it's too generic)